docs/Contract Creators/l2-sequencer-flag.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: nodes.liquid
+date: Last Modified
+title: "L2 Sequencer Health Flag Docs"
+permalink: "docs/l2-sequencer-flag/"
+---
+
+The idea behind an Optimistic Rollup (OR) type of protocol is to move all execution off-chain and keep all transaction data available on-chain. Such protocols have a special off-chain component, a Sequencer, that executes and rolls up the Layer 2 transactions by batching multiple transactions into a single one.
+
+If a sequencer becomes unavailable, it becomes impossible to access read/write APIs that consumers are using so every dapp will be down for 95% of the users, except those that know how to interact with the Layer 1 OR contracts. In this case, it would be unfair to continue providing service on your dapp, as only 5 % of the users can use it. Note, this doesn't mean that the Layer 2 chain has stopped, as OR is not an actual chain. 
```

Comment:
"Note, this doesn't mean that the Layer 2 chain has stopped, as OR is not an actual chain. " Doesn't quite make sense. We say it's a chain in the first sentence and then say it isn't in the second?